packages/components/src/components/text-input/_text-input.scss

```diff
@@ -38,12 +38,19 @@
     &:active {
       @include focus-outline('outline');
     }
-
+    &-wrapper-inline {
```

Comment:
Since this is a modifier class, can we rename this to `--inline`?

Also, we are trying not to nest selectors as we move forward. Can we unnest this selector and write it like

```scss
.#{$prefix}--text-input-wrapper--inline {
   flex-direction: row;
}
```

Reply:
Thanks for your comments. I will update.